src/lab/Introduction.html

```diff
@@ -40,7 +40,7 @@
 			                    <span class="icon-bar" style="background-color: #77BB41"></span>
 			                    <span class="icon-bar" style="background-color: #77BB41"></span>
 			                </button>
-			                <a class="navbar-brand main-logo-a" href="http://vlabs.ac.in/index.html"><img src="../../images/logo-new.png" class="img-responsive main-logo" /></a>
```

Comment:
1,2,3,4,5 specify that logo link has been changed in Introduction,Index,list of experiments,Courses,Target